pkg/issuer/acme/dns/util/wait.go

```diff
@@ -98,7 +108,7 @@ func checkDNSPropagation(fqdn, value string, nameservers []string) (bool, error)
 // checkAuthoritativeNss queries each of the given nameservers for the expected TXT record.
 func checkAuthoritativeNss(fqdn, value string, nameservers []string) (bool, error) {
 	for _, ns := range nameservers {
-		r, err := dnsQuery(fqdn, dns.TypeTXT, []string{net.JoinHostPort(ns, "53")}, false)
+		r, err := dnsQuery(fqdn, dns.TypeTXT, []string{ns}, true)
```

Comment:
The test failure is probably because of the change from non-recursive to recursive here, but we're asking an authoritative server, so it should make no difference

Reply:
Could the failure be because we've gone from `net.JoinHostPort(ns, "53")` to `ns`? 😄 perhaps `dnsQuery` requires `8.8.8.8:53` instead of just `8.8.8.8`

Reply:
@munnerz spot on, will get it fixed

Reply:
nevermind, it's because of the change to the nameservers strings. Should work again if you put ":53" at the end of each nameserver in the wait_test.go file

Reply:
Should be fixed up now.